api_spec.md

```diff
@@ -0,0 +1,141 @@
+# APIs
+
+## Create user
+- Method: POST
+- Endpoint: /passenger/registers
+- Request:
+   - Request body of 
+     - full_name
+     - full_last_name
```

Comment:
Full name implies that it's first and last names combined, if you want to have them separate then rename to `first_name` and `last_name`